src/crimson/net/ProtocolV2.cc

```diff
@@ -1384,8 +1384,9 @@ ceph::bufferlist ProtocolV2::do_sweep_messages(
 
     auto message = MessageFrame::Encode(header2,
         msg->get_payload(), msg->get_middle(), msg->get_data());
-    logger().debug("{} write msg type={} off={} seq={}",
-                   conn, header2.type, header2.data_off, header2.seq);
+    logger().info("{} --> [{} {}] #{} === {} ({}) // {}",
+		  messenger, peer_name, conn.get_peer_addr(),
+		  msg->get_seq(), *msg, msg->get_type(), conn);
```

Comment:
`info` logs will be printed by default, so this means we need to change default log level to >= `warn` for perf tests.

Reply:
@cyx1231st i see. how about moving all the logging messages in messenger to trace level ?

Reply:
Agree, crimson messenger haven't utilized trace level logs before.